packages/core-mobile/app/contexts/DeeplinkContext/types.ts

```diff
@@ -30,12 +30,10 @@ export enum StakeActions {
 
 export const ACTIONS = {
   WC: 'wc',
-  StakeComplete: StakeActions.StakeComplete
+  StakeComplete: StakeActions.StakeComplete,
+  OpenChainPortfolio: 'openchainportfolio'
```

Comment:
nit: maybe use camel case instead?

Reply:
StakeComplete is the same, i guess the reason is because this string ends up in deeplink url core://oenchainportfolio